Sources/Display/Platform/Win32/win32_window.cpp

```diff
@@ -1685,8 +1735,6 @@ void Win32Window::create_hid_devices()
 
 void Win32Window::get_styles_from_description(const DisplayWindowDescription &desc, DWORD &style, DWORD &ex_style)
 {
-	style = 0;
-	ex_style = 0;
```

Comment:
Actually I meant those arguments as purely out arguments rather than in/out. I really wish C++ had real qualifiers that could indicate this, or easily support multiple return arguments.

About the caller also initializing the fields, that is a coding practice I always use to ensure no variable is ever left in an uninitialized state. Basically both places function as a guard in case one of them failed to clear the variables before use.